MREUnityRuntime/MREUnityRuntimeLib/API/MREApi.cs

```diff
@@ -139,10 +139,11 @@ public class MREAppsAPI
 		/// </summary>
 		/// <param name="ownerScript">The owner unity script for the app.</param>
 		/// <param name="globalAppId">The global app id for the app being instanced, or empty if there is no global app id.</param>
+		/// <param name="ephemeralAppId">A string uniquely identifying the MRE instance on all clients.</param>
 		/// <returns>Returns the newly created mixed reality extension app.</returns>
-		public IMixedRealityExtensionApp CreateMixedRealityExtensionApp(MonoBehaviour ownerScript, string globalAppId = "")
+		public IMixedRealityExtensionApp CreateMixedRealityExtensionApp(MonoBehaviour ownerScript, string ephemeralAppId, string globalAppId = "")
```

Comment:
The default value for the global app ID should be removed now that we have these two ID's. Both are really required in some kind of fashion if a host app wants to allow any kind of user tracking.